modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -134,6 +134,21 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 // CCFR here.
 
+=== Directly right clicking on a `<ol>`/ `<li>` in FireFox didn't enable the button `List Properties...` in the context menu
+// #TINY-10490
+
+In previous versions of {productname}, two issues where identified that affected `list properties`;
+
+. in old versions of Firefox (prior to version 121), the caret was not moved to the target position, and;
```

Comment:
maybe specify that the caret was not moved to the target position using the `right click`

Reply:
Fixed. Also made a change to the cause paragraph to specify that this issue only affects <ol><li> tags and the context menu item being enabled/disabled. Resolve this conversation if you are happy with the changes.